examples/CapabilityStatement/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
The `.gitignore` should be at the root of the project

Reply:
I'd be very happy to hear your opinion on that. I used to put .gitignore files in subfolders so that the folders would still be pushed into the repository even if you don't want its contents to be pushed. 
Is there another way to do it?

Reply:
Do you mean that you want to push an empty folder to the repo? I'm not sure why we'd like to do so

Reply:
Yeah, that's exactly what I was thinking!
It's true that looking at the code, here it's not necessarily essential (it's only for the example notebook). I'm gonna move the gitignore  :) .

In general, I liked to do that for subjects with a lot of data in which we have folders where we store data (inputs, outputs, etc.). I find that it allows you to quickly understand how the project is organized thanks to its tree structure, ie quickly understand where you should put the file with the "confidential" input data (for example).